game-warhammer40kdarktide/index.js

```diff
@@ -123,37 +288,21 @@ async function serializeLoadOrder(_context, loadOrder) {
     { encoding: "utf8" }
   );
 }
-
-let supportedTools = [
-  {
-    id: "dtkit-patch",
-    name: "Toggle modding (patch/unpatch game files)",
-    executable: () => "tools/dtkit-patch.exe",
-    requiredFiles: ["tools/dtkit-patch.exe"],
-    queryPath,
-  },
-];
+//end source code github
 
 function main(context) {
+  context.registerInstaller('tetstidk-mod', 25, testSupportedContent, installContentverifier);

```

Comment:
What is this? What do tetstidk-mod' and 25 mean?

Reply:
tetstidk-mod is a name to indicate to Vortex what it is (I don't really understand but it can be changed and it should be changed).
25 is the priority vortex uses for modifications ,installing etc.

Reply:
Got it, 25 is a default suggestion installer priority https://wiki.nexusmods.com/index.php/Creating_a_game_extension_for_Vortex

The first one is a name. Maybe we should change it to somethning like `warhammer40kdarktide-mod`

https://wiki.nexusmods.com/index.php/Creating_a_game_extension_for_Vortex